uui-docs/src/peContexts/TabButtonContext.tsx

```diff
@@ -1,13 +1,13 @@
 import * as React from 'react';
 import { DemoComponentProps } from '../types';
 import { ButtonProps } from '@epam/uui-components';
-import { Panel, FlexRow, TabButtonMods } from '@epam/uui';
+import { Panel, FlexRow, TabButtonProps } from '@epam/uui';
 
 interface DemoComponentState {
     activeTab: 'Main' | 'demoTab' | 'Tools' | 'Options';
 }
 
-export class TabButtonContext extends React.Component<DemoComponentProps<ButtonProps & TabButtonMods>, DemoComponentState> {
+export class TabButtonContext extends React.Component<DemoComponentProps<ButtonProps & TabButtonProps>, DemoComponentState> {
```

Comment:
I guess we need only `TabButtonProps`, because they already contains all from `ButtonProps`